libs/agno/agno/models/base.py

```diff
@@ -990,21 +990,22 @@ def _show_tool_calls(self, function_calls_to_run: List[FunctionCall], model_resp
         """
         Show tool calls in the model response.
         """
+        # Only add to content for regular text responses
         if len(function_calls_to_run) == 1:
             if model_response.content and len(model_response.content) > 0 and model_response.content[-1] != "\n":
                 model_response.content += "\n\n"
             else:
                 model_response.content = ""
-            model_response.content += f" - Running: {function_calls_to_run[0].get_call_str()}\n\n"
+            model_response.content += f"Running: \n\n • {function_calls_to_run[0].get_call_str()}\n\n"
```

Comment:
Should we not already here store it on a different field on the model_response? Then you don't have to check for tool_calls, you can just check for that field. That could be `formatted_tools`

Reply:
Oh wait I see what you did, in that case should we not remove this completely?

Reply:
oh yes, i can just do 
```py
def _show_tool_calls(self, function_calls_to_run: List[FunctionCall], model_response: ModelResponse):
        """
        Show tool calls in the model response.
        """
        pass
```

that also works, since we no longer need to append the tool call info in response content